packages/augur-ui/src/modules/common/progress.tsx

```diff
@@ -209,13 +209,13 @@ export const CountdownProgress = (props: CountdownProgressProps) => {
 
     timeLeft = time.timestamp - currentTime.timestamp;
     countdown = (countdownBreakpoint || OneWeek) >= timeLeft && timeLeft > 0;
-    valueString = countdown
-      ? `${daysRemaining}:${
+    valueString = !countdown || label === 'Expired'
```

Comment:
Dont use Expired, 
If the market is awaiting finalization or finalized then there is no countdown timer.